src/interval.jl

```diff
@@ -150,12 +150,50 @@ end
 
 ##### EQUALITY #####
 
-function Base.isless(a::AbstractInterval{T}, b::AbstractInterval{T}) where T
+Base.:<(a::AbstractInterval{T}, b::T) where T = RightEndpoint(a) < b
+Base.:<(a::T, b::AbstractInterval{T}) where T = a < LeftEndpoint(b)
+
+function Base.:<(a::AbstractInterval{T}, b::AbstractInterval{T}) where T
+    return LeftEndpoint(a) < LeftEndpoint(b)
+end
+
+function disjoint_less_than(a::AbstractInterval{T}, b::AbstractInterval{T}) where T
     return RightEndpoint(a) < LeftEndpoint(b)
 end
```

Comment:
Should we implement a version of this for scalars and intervals?

Reply:
I went back and forth about that a fair bit. `scalar < interval` is always going to mandate disjoint, while `interval < scalar` can have a meaningful distinction here.

Probably makes sense to implement both.